Traceback (most recent call last):
  File "/Users/Joo/Desktop/Development/Python/Reference/boring_stuff/c10_debugging/notes.py", line 113, in <module>
    raise Exception('This is the error message.')
Exception: This is the error message.
